plugins/userBalance.ts

```diff
@@ -0,0 +1,9 @@
+export default ({ store }): void => {
+  const account = store.getters.getAuthAddress
+  if (account) {
+    store.dispatch('fetchBalance', account)
```

Comment:
Why you need both fetch and subscribe?

Reply:
@vikiival I think for the first time subscribe will not return balance, it will only work in case of any balance changes, but let me confirm once more.